### ISSUE TITLE
Empty screen

### ISSUE THREAD

**Issue by Wulfheart** [LarsWiegers/laravel-maps#3]
Fresh Laravel app.
View:
```html
<!DOCTYPE html>
<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
    </head>
    <body class="antialiased">
    <x-maps-leaflet></x-maps-leaflet>
    </body>
</html>
```

Gives the following html:
```html
<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
    </head>
    <body class="antialiased">
    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css"
      integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A=="
      crossorigin=""/>

<style>
    #defaultMapId {
        height: 100vh;
    }
</style>

<div id="defaultMapId"></div>

<!-- Make sure you put this AFTER Leaflet's CSS -->
<script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"
        integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA=="
        crossorigin=""></script>
<script>

    var mymap = L.map('mapid').setView([0, 0], 13);
    
            let url = 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png';
        L.tileLayer(url, {
        maxZoom: 18,
        attribution: 'Map data &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, ' +
            'Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
        id: 'mapbox/streets-v11',
        tileSize: 512,
        zoomOffset: -1
    }).addTo(mymap);
</script>
    </body>
</html>

```

But renders as this:
![image](https://user-images.githubusercontent.com/25671390/120626862-62017080-c463-11eb-82dc-0ba6274c836b.png)

**Comment by LarsWiegers** [comment]
Hi, 
Thanks for creating an issue when you encountered a bug! ( You are actually the first real issue so congrats 🎉).
Let me look in to it and ill be right back.

**Comment by LarsWiegers** [comment]
I released a new version 0.2 that should be downloadable soon( packagist has to update first ). This seems to work for me with your code.  @Wulfheart  please let me know if it also works on your side :)

**Comment by Wulfheart** [comment]
Awesome. Works on my end!
